package.json

```diff
@@ -7,6 +7,9 @@
   "bugs": {
     "url": "https://github.com/asyncapi/modelina/issues"
   },
+  "engines": {
+    "node": "14"
```

Comment:
You should use the `>=14` version. It perhaps hardcodes the given version to the 14.

Reply:
Ah, right 👍